mrp_production_estimated_cost/wizard/wiz_create_fictitious_of.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# (c) 2014-2015 Avanzosc
+# (c) 2014-2015 Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields, api
```

Comment:
Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports